tutorial/markdown/nodejs/express.md

```diff
@@ -380,21 +371,75 @@ We have created a start script in our `package.json` that executes babel-node an
 },
 ```
 
-There are other scripts that we have added for our maintenance and testing of the project on GitHub using GitHub actions, a lot of this is above and beyond the scope of this quickstart, and can be done in many different ways, but we wanted to include as much as we can to help you on your journey as far as configuring a project for use with JavaScript and in our case ES6 using Babel and basic testing and CI setup.
+There are other scripts that we have added for our maintenance and testing of the project on GitHub using GitHub actions.
 
 ```json
   "scripts": {
-    "init-test-db": "env-cmd ./config/test.env npx babel-node ./src/initializeCbServer.js",
-    "init-test-index": "env-cmd ./config/test.env npx babel-node ./src/createIndex.js",
-    "test": "env-cmd ./config/test.env jest --verbose",
-    "test:ci": "npm run init-test-db && npm run init-test-index && env-cmd ./config/test.env jest --verbose",
-    "init-db": "env-cmd ./config/dev.env npx babel-node ./src/initializeCbServer.js",
-    "start": "env-cmd ./config/dev.env nodemon --exec babel-node src/server",
+    "start": "env-cmd -f ./config/dev.env nodemon --exec babel-node src/server",
+    "test": "env-cmd -f ./config/test.env jest --verbose --forceExit --detectOpenHandles",
+    "testGA": "jest --verbose --runInBand",
     "clean": "rm -rf dist",
     "build": "npm run clean && babel ./src --out-dir dist --copy-files"
   },
 ```
 
-## Conclusion
+## Running Tests
+
+We have defined integration tests using [jest](https://jestjs.io/) for all the API end points. The integration tests use the same database configuration as the application. After the tests, the documents are cleaned up.
+
+The tests are configured in the `__test__` folder.
+
+To run the tests, run the below command after copying the content of the `dev.env` file to the `test.env` to ensure it uses the same database configuration:
+
+```bash
+# Execute this command in the project's root directory
+npm run test
+```
+
+## Appendix
+
+### Extending API by Adding New Entity
+
+If you would like to add another entity to the APIs, these are the steps to follow:
+
+- **Create the New Entity in Couchbase Bucket:**
+  - Utilize the [Couchbase Server interface](https://docs.couchbase.com/cloud/n1ql/n1ql-language-reference/createcollection.html) to establish the new collection within the Couchbase bucket.
+
+- **Define the New Route:**
```

Comment:
Can you also add the reference to how you can do it via the SDK?
https://docs.couchbase.com/sdk-api/couchbase-node-client/classes/CollectionManager.html

Reply:
Added alternative way to create collection via sdk